MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+recursive-include collective *
+
+include README.md
```

Comment:
Should this be README.txt?
I would run check-manifest to verify this file is ok :)

Reply:
Good catch, and thanks for the pointer! TIL about `check-manifest`.

I pushed a fix.